apimonitor/monitor/health.py

```
@@ -0,0 +1,162 @@
+                    'service': service.name,
+                    'url': url,
+                    'error': 'network',
+                    'details': str(e),
+                    'attempt': attempt + 1,
+                    'ts': time.time()
+                }
+            )
+            
+        except HTTPStatusError as e:
+            # This shouldn't happen since we handle all status codes
+            last_error = f"HTTP error: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)